accepted/future-releases/generalized-typedef-2018/implementation-plan.md

```diff
@@ -62,14 +62,35 @@ where `type` can be any type, rather than just a function type.
 All tools implement support for using such type aliases, in all situations
 mentioned under phase 0.
 
-### Phase 2 (Release)
+### Phase 2 (IDE tooling and Documentation)
 
-The feature is released as part of the next stable Dart release.
+IDEs and other tools which consume/display Dart code should be updated to handle
+the new syntax.
 
-## Timeline
+IDE support for any desired quick fixes, refactors, go-to-definition, etc should
```

Comment:
@devoncarew @bwilkerson do you want to add specific goals here in a follow on PR, or track them elsewhere?

Reply:
cc @scheglov who's done much of the work here. I'm not sure how far we are through the implementation plan (go/generalized-typedefs-tooling-plan) and what additional work we'd want to track in issues

Reply:
We have support for navigation, semantic highlighting, completion, outline, search and rename, hover.

I don't have plans to do extract/inline type alias refactorings.
I'm not aware about any specific quick assists or fixes.

Reply:
We'll open issues for other generalized typedef work we want track.